zgw_consumers/admin.py

```diff
@@ -16,6 +17,11 @@ class ServiceAdmin(admin.ModelAdmin):
     list_display = ("label", "api_type", "api_root", "nlx", "auth_type")
     list_filter = ("api_type", "auth_type")
     search_fields = ("label", "api_root", "nlx", "uuid")
+    readonly_fields = ("get_connection_check",)
+
+    @admin.display(description=_("Connection check status code"))
+    def get_connection_check(self, obj):
+        return obj.connection_check
```

Comment:
I'm still not seeing tests that actually render this field in the admin:

* opening the service change page without configuring the field
* opening the service change page with configured field
* opening the service add page

So far you're only testing the model property, but there may be edge cases that cause the admin to crash (e.g. if `obj` is `None`)